dom0/sd-sys-vms.sls

```diff
@@ -117,18 +119,12 @@ sd-{{ sys_vm }}-fedora-version-update:
       - qvm: create-{{ sd_supported_fedora_template }}
 {% endif %}
 
-# We're numbering our VMs now even though its contents wouldn't change, to work
-# around Salt's limitations, so the non-numbered VM should be removed.
-{% if sys_vm == "sys-usb" %}
-remove-sd-fedora-dvm:
-  qvm.absent:
-    - name: sd-fedora-dvm
-    - require:
-      - qvm: sd-sys-usb-fedora-version-update
-
 # Finally, remove the old supported fedora DVM we created. We won't uninstall
 # the template, in case it's being used elsewhere, but the `sd-` VMs we can
 # reasonably manage (remove) ourselves.
+# TODO: does this need to be updated to refer to sd-fedora-<previous>-dvm? or was it
+# a one-time thing?
```

Comment:
Yes, if we're still going with `sd-fedora-xx-dvm` for the sys-usb dvm template, it would be friendly to clean up the old ones when we're not using them anymore.  But since we're installing clean on 4.2, users won't have an out-of-date sd-fedora dvm on their machines, so this won't be relevant til the next fedora base template upgrade.